src/services/discord/discord.service.ts

```diff
@@ -1,28 +1,26 @@
 // Initializes the `discord` service on path `/discord`
-import { ServiceAddons } from '@feathersjs/feathers';
-import { Application } from '../../declarations';
-import { Discord } from './discord.class';
-import createModel from '../../models/discord.model';
-import hooks from './discord.hooks';
+import { ServiceAddons } from "@feathersjs/feathers";
+import { Application } from "../../declarations";
+import { Discord } from "./discord.class";
+import createModel from "../../models/discord.model";
+import hooks from "./discord.hooks";
```

Comment:
This file still has double-quotes. I think we can fix this by running `npm run lint`. Later we could automate this in case we forget to run it 👍

Reply:
Alright I will run the lint command. Thanks for your feedback.